README.md

```diff
@@ -1301,6 +1310,203 @@ By default, it will revert all the clusters present in the backup folder
+using “*--profile \<profile_name\>*” or “*-p \<profile_name\>*”. You can
+test that the correct workspace is getting selected by running any
+Databricks CLI command. For example, you can run “*databricks clusters
+list -p prod*” and check that the Prod clusters are being returned. Once
+the profile is verified, you can run UCX install for that specific
+profile: “*databricks labs install ucx -p prod*”.
+
+**Account Level:** Multiple account level profiles are set in the
+.databrickscfg file.
+
+**Solution:** The installation command “*databricks labs install ucx*”
+will provide an option to select one account profile.
+
+### Authentication Issues
+
+**Account Level:** Not only workspace, but account level authentication
+is also needed for installing UCX. If you do not have an account
+configured in .databrickscfg, you will get an error message
+“.databrickscfg does not contain account profiles; please create one
+first”.
+
+**Solution:** To authenticate with a Databricks account, consider using
+one of the following authentication types: [<u>OAuth machine-to-machine
+(M2M)
+authentication</u>](https://docs.databricks.com/en/dev-tools/cli/authentication.html#m2m-auth),
+[<u>OAuth user-to-machine (U2M)
+authentication</u>](https://docs.databricks.com/en/dev-tools/cli/authentication.html#u2m-auth),
+[<u>Basic authentication
+(legacy)</u>](https://docs.databricks.com/en/dev-tools/cli/authentication.html#basic-auth).
+
+### Workspace has an external Hive Metastore (HMS)
+
+**External HMS connectivity from UCX clusters:** If the workspace has an
+external HMS, the clusters running the UCX jobs need to have specific
+configurations to connect to the external HMS. Otherwise, UCX assessment
+will not be able to assess the tables on HMS.
+
+**Solution:** Use a cluster policy before installation to set the
+required Spark config for connecting to the external HMS, or manually
+edit the cluster post-installation to have the correct configurations.
+Detailed steps can be found
+[<u>here</u>](https://github.com/databrickslabs/ucx/blob/main/docs/external_hms_glue.md).
+
+**External HMS connectivity from UCX SQL warehouse:** UCX requires a SQL
+warehouse to create tables, run queries, create and refresh dashboards.
+If you already have a Pro or Serverless warehouse connected to the
+external HMS, you can select the same warehouse for UCX. You will also
+be given an option (during installation) to create a new warehouse for
+UCX. If you have never used a warehouse before, the new warehouse
+created might not have proper configuration set to connect to the
+external HMS.
+
+**Solution:** Set Spark configuration for connecting to external HMS in
+the Admin Settings of SQL warehouse. This will only be needed if the
+admin settings do not have the configurations already set. For example,
+add *spark.hadoop.javax.jdo.option.ConnectionURL \<connectionstring\>*
+under Data Access Configuration of SQL Warehouse Admin Settings.
+
+# Verify the Installation
```

Comment:
why does it have to be top-level header? ### is consistent with the previous.

Reply:
Done